hypervisor/arch/x86/lapic.c

```diff
@@ -413,18 +413,16 @@ send_startup_ipi(enum intr_cpu_startup_shorthand cpu_startup_shorthand,
 	return status;
 }
 
-void send_single_ipi(uint16_t pcpu_id, uint32_t vector)
+/* dest must be INTR_LAPIC_ICR_PHYSICAL(0x0U) or INTR_LAPIC_ICR_LOGICAL(0x1U) */
+void send_dest_ipi(uint32_t dest_mask, uint32_t vector, uint32_t dest)
```

Comment:
The 1st parameter is actually dest, not mask.
The last parameter is unknown to me.

Reply:
1st is dest_mask which is destination filed, last parameter is destination mode. I will update the name.

Reply:
then use dest and del_mode?

Reply:
Will use dest and dest_mode.